client/my-sites/customer-home/cards/education/find-success/index.jsx

```diff
@@ -0,0 +1,30 @@
+import { useTranslate } from 'i18n-calypso';
+import React from 'react';
+import findSuccessPrompt from 'calypso/assets/images/customer-home/illustration--secondary-find-success.svg';
+import { EDUCATION_FIND_SUCCESS } from 'calypso/my-sites/customer-home/cards/constants';
+import EducationalContent from '../educational-content';
+
+const FindSuccess = () => {
+	const translate = useTranslate();
+
+	return (
+		<EducationalContent
+			title={ translate( '10 ways to find success with your website' ) }
+			description={ translate(
+				"You need to keep traffic coming to your site so you get leads and sales. We'll teach you know"
```

Comment:
I think there's a typo here!

```suggestion
				"You need to keep traffic coming to your site so you get leads and sales. We'll teach you how."
```